### ISSUE TITLE
Why the Facial Recognition Model I trained using the method you provided didn't work? 

### ISSUE THREAD

**Issue by iDTer** [jchaykow/AGN-pytorch#3]
As [article](https://towardsdatascience.com/finetune-a-facial-recognition-classifier-to-recognize-your-face-using-pytorch-d00a639d9a79) mentioned,  we need build a facial recognition classifier first. 

![image](https://user-images.githubusercontent.com/30014931/190190960-42c627cb-fa91-4980-8fcd-642eb2c71f59.png)

But as the picture shows, the FRT Loss didn't drop during training. 

Thank you for your time and attention. I would appreciate it very much if you could give me a reply.

**Comment by iDTer** [comment]
This is still the case when I increase the epoch to 500.
